sympy/physics/continuum_mechanics/truss.py

```diff
@@ -0,0 +1,494 @@
+        >>> t.add_member('AB', 'A', 'B')
+        >>> t.add_member('AC', 'A', 'C')
+        >>> t.add_member('BC', 'B', 'C')
+        >>> t.members
+        [['AB', 'A', 'B'], ['AC', 'A', 'C'], ['BC', 'B', 'C']]
+        >>> t.remove_member('AC')
+        >>> t.members
+        [['AB', 'A', 'B'], ['BC', 'B', 'C']]
+        """
+        if label not in self._member_labels:
+            raise ValueError("No such member exists in the Truss")
+
+        else:
+            self._members.remove([label, self._member_nodes[label][0], self._member_nodes[label][1]])
+            self._member_labels.remove(label)
+            self._nodes_occupied.pop(tuple([self._member_nodes[label][0], self._member_nodes[label][1]]))
+            self._nodes_occupied.pop(tuple([self._member_nodes[label][1], self._member_nodes[label][0]]))
+            self._member_nodes.pop(label)
+            self._internal_forces.pop(label)
+
+    def change_label(self, label, new_label):
+        """
+        This method changes the label of a node or a member.
+
+        Parameters
+        ==========
+        label: String or Symbol
+            The label of the node/member for which the label has
+            to be changed.
+
+        new_label: String or Symbol
+            The new label of the node/member.
+
+        Examples
+        ========
+
+        >>> from sympy.physics.continuum_mechanics.truss import Truss
+        >>> t = Truss()
+        >>> t.add_node('A', 0, 0)
+        >>> t.add_node('B', 3, 0)
+        >>> t.nodes
+        [('A', 0, 0), ('B', 3, 0)]
+        >>> t.change_label('A', 'C')
+        >>> t.nodes
+        [('C', 0, 0), ('B', 3, 0)]
+        >>> t.add_member('BC', 'B', 'C')
+        >>> t.members
+        [['BC', 'B', 'C']]
+        >>> t.change_label('BC', 'BC_new')
+        >>> t.members
+        [['BC_new', 'B', 'C']]
+        """
+        if label not in self._node_labels and label not in self._member_labels:
+            raise ValueError("No such node/member exists")
+        if label in self._node_labels:
+            for node in self._nodes:
+                if node[0] == label:
+                    self._node_labels[self._node_labels.index(node[0])] = new_label
+                    self._loads[new_label] = self._loads[node[0]]
+                    self._loads.pop(node[0])
+                    self._supports[new_label] = self._supports[node[0]]
+                    self._supports.pop(node[0])
+                    if self._supports[new_label] == 'pinned':
+                        self._reaction_forces['R_'+str(new_label)+'_x'] = self._reaction_forces['R_'+str(label)+'_x']
+                        self._reaction_forces['R_'+str(new_label)+'_y'] = self._reaction_forces['R_'+str(label)+'_y']
+                    elif self._supports[new_label] == 'roller':
+                        self._reaction_forces['R_'+str(new_label)+'_y'] = self._reaction_forces['R_'+str(label)+'_y']
+                    for member in self._members:
```

Comment:
The old label's reaction forces should be removed as well, currently it shows both the old label and the new label reaction force when `self.reaction_forces` is called.

Also this whole section might be changed after you fix the apply_support function, you  might have to remove the existing label reaction loads by using `remove_load` and apply new loads as well.